contrib/devtools/README.md

```diff
@@ -89,66 +89,6 @@ example:
 BUILDDIR=$PWD/build contrib/devtools/gen-manpages.sh
 ```
 
-github-merge.py
```

Comment:
Let's add this documentation to the maintainer tools repo too so it doesn't get lost.

Reply:
I've got https://github.com/bitcoin-core/bitcoin-maintainer-tools/pull/29 open to do just that 👍